content/fq_FilterEditor.js

```diff
@@ -412,11 +377,152 @@
   }
 
   customElements.define("filtaquilla-ruleactiontarget-javascriptaction", FiltaQuillaRuleactiontargetJavascriptAction);
-    
-  util.logDebug("filterEditorOverjay.js - Finished.");
-    
-    
-})();
 
+  util.logDebug("fq_FilterEditor.js - Finished.");
+
+
+}
+
+// // XXX: As long as CSS "-moz-binding" works, the following code doesn't have
```

Comment:
This large commented-out block of code is to remind us, that it may be necessary at some point (when "-moz-binding" will stop work for "search-value-xxx" elements as it happened for "ruleactiontarget-xxx" elements).